client/components/theme-upgrade-modal/index.tsx

```diff
@@ -370,6 +432,12 @@ export const ThemeUpgradeModal = ( {
 		featureListHeader = translate( 'Included with your %(businessPlanName)s plan', {
 			args: { businessPlanName: plans.data?.[ PLAN_BUSINESS ]?.productNameShort || '' },
 		} );
+	} else if ( theme?.data?.theme_tier?.feature === FEATURE_PERSONAL_THEMES ) {
```

Comment:
We need to change this if with a feature check after testing and before merging.